.travis.yml

```diff
@@ -4,22 +4,24 @@ python:
   - 3.3
   - 3.4
   - 3.5
-#  - 3.6
+  - 3.6
 env:
   - DJANGO=1.7
   - DJANGO=1.8
   - DJANGO=1.9
-#  - DJANGO=1.10
-#  - DJANGO=1.11
+  - DJANGO=1.10
+  - DJANGO=1.11
 matrix:
   exclude:
-    - python: 3.5
-      env: DJANGO=1.7
-    - python: 3.3
-      env: DJANGO=1.9
+    # Django 1.7 does not support Python 3.5+
+    - { python: 3.5, env: DJANGO=1.7 }
+    - { python: 3.6, env: DJANGO=1.7 }
+    # Django 1.9+ does not support Python 3.3
+    - { python: 3.3, env: DJANGO=1.9 }
+    - { python: 3.3, env: DJANGO=1.10 }
+    - { python: 3.3, env: DJANGO=1.11 }
   include:
-    - python: 2.7
-      env: TOXENV=flake8
+    - { python: 2.7, env: TOXENV=flake8 }
```

Comment:
I compacted these so that they're one per line, and hopefully more readable. Does that look good?

Reply:
Not bad! Though my eyes feel irritated with curly braces in a YAML file!  :nerd_face: 

I'm still shocked that you care so much about those comments. Here comes my okay: :+1: :confounded:

Reply:
Maybe it's not all that bad ... :smiley: ... in combination with the quirky curly braces!

Thanks for not giving up on us!  :1st_place_medal: